docker/build.sh

```diff
@@ -1,6 +1,14 @@
 set -e
+
+STATIC_LINKING=${STATIC_LINKING:-OFF}
+RUN_TESTS=${RUN_TESTS:-1}
+
 mkdir -p "$1"
 cd "$1"
-cmake -DCMAKE_BUILD_TYPE="$2" -DSTATIC_LINKING:BOOL=ON ../
```

Comment:
@ajor docs still say that:

> Building bpftrace using the Docker method below will always result in a statically linked executable.

* https://github.com/iovisor/bpftrace/blob/master/INSTALL.md#compilation